proto/resource_usage_agent.proto

```diff
@@ -14,21 +14,27 @@ option java_package = "org.tikv.kvproto";
 
 // ResourceUsageAgent is the service for storing resource usage records.
 service ResourceUsageAgent {
-    // Report the CPU time records. By default, the records with the same
+    // Report the resource usage records. By default, the records with the same
     // resource group tag will be batched by minute.
-    rpc ReportCPUTime(stream CPUTimeRecord) returns (EmptyResponse) {}
+    rpc Report(stream ResourceUsageRecord) returns (EmptyResponse) {}
```

Comment:
Change the RPC name should be a change brokes compatibility. Not sure if we really want it.

Reply:
> Change the RPC name should be a change brokes compatibility. Not sure if we really want it.

It seems that if the RPC interface is integrated, the method name needs to be changed (it better be), and there is a break change.
Please help confirm whether we can do this, or add a new method to make the transition. /cc @breeswish @zhongzc

Reply:
Yes, changing RPC name is a breaking change.

Reply:
How about leaving `ReportCPUTime`, but mark it as deprecated? Then we can add a new interface `Report`.

New clients should never call `ReportCPUTime` any more. But current receivers should accept both `ReportCPUTime` and `Report`.

Reply:
Fine, this is the most suitable way.